Userland/Applications/PixelPaint/Layer.cpp

```diff
@@ -237,12 +237,12 @@ ErrorOr<void> Layer::crop(Gfx::IntRect const& rect, NotifyClients notify_clients
     return {};
 }
 
-ErrorOr<void> Layer::resize(Gfx::IntSize new_size, Gfx::IntPoint new_location, Gfx::Painter::ScalingMode scaling_mode, NotifyClients notify_clients)
+ErrorOr<void> Layer::resize(Gfx::IntRect const& new_rect, Gfx::Painter::ScalingMode scaling_mode, NotifyClients notify_clients)
```

Comment:
still a misnomer here. still i havenʼt found a decent name. could describe what that method does exactly in your own words (given the arguments)? 🤔

(iʼm just concerned with layer changing the position during the resize—do we really need a location update in `L271`? if not then we could go back to size as a parameter…?)

Reply:
I've renamed `Layer::resize()` has been renamed to `Layer::scale()`. I think this more accurately describes the transform being performed and is consistent with methods that perform a similar function elsewhere in the codebase.